lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -517,6 +519,18 @@ def verify_paging_flow_rules(self, ip_list):
             )
             assert bool(has_tunnel_action)"""
 
+    def gen_imsi(self, prefix=None):
```

Comment:
maybe throw an exception if the overall length of  idx + prefix is longer than IMSI_LEN.